MANIFEST.in

```diff
@@ -1 +1,2 @@
-include queries/*.txt
\ No newline at end of file
+include queries/*.txt
+include templates/*.nii
```

Comment:
@TomMaullin: Usually I would not recommend having NIfTI files in a GitHub repository (because binary files are not handled well). But, because those files will likely not change much in the future, it could be okay to have them... 

 1. Can you check how much space the templates folder is taking up? (I think GitHub repo should be no more than 1GB, including all versions).
 2. Can you check under which licence those NIfTI template images are distributed?

Reply:
Yes, I had the same worry, currently they take up 2.7Mb but the T1 can be removed if necessary. I used the T1_skullstripped version for the viewer but created it myself using the T1 (the skull in the T1 sometimes ends up cropped in the slicer view). 

The T1 image was taken from the OldNorm toolbox in SPM so I guess this means it is under the SPM GNU public license? Do you think this is okay?

Reply:
Yes, please remove as much as you can to have a minimal number of NIfTI images in the repo.

As far as I know, GPL can be used to licence code but not data. I checked [spm_templates.man](https://github.com/spm/spm12/blob/master/spm_templates.man) but could not find info on licenses for the templates... 

One option: could you automatically download the template (when needed) if it's not available locally, rather than embed the image in the repo? E.g. from https://github.com/spm/spm12

Reply:
The issue I would have with automatically downloading from a public repo is that the template I was using is the one I had manually skull-stripped (as the one with the skull included is cropped by FSL slicer when making slice images). I can't download this template from that repository. What do you think is best? Perhaps I could download it and then skull strip it? This seems a bit much though...

Reply:
How about either 1) using the non-skull-stripped version or 2) doing the skull-stripping after download (only once)?